Project-I-Social-Card/social-card/src/components/HeaderComponents/ImageThumbnail.js

```diff
@@ -1,2 +1,14 @@
 import React from 'react';
 import './Header.css';
+
+
+const ImageThumbnail = () => {
+
+	return (
+		<div className="logo">
+			<img src="https://tk-assets.lambdaschool.com/1c1b7262-cf23-4a9f-90b6-da0d3c74a5c6_lambdacrest.png"/>
```

Comment:
in general, if you div only contains one element you probably don't even need it.